tensorboard/plugins/histogram/summary_v2.py

```diff
@@ -270,8 +277,10 @@ def histogram_pb(tag, data, buckets=None, description=None):
     """
     bucket_count = DEFAULT_BUCKET_COUNT if buckets is None else buckets
     data = np.array(data).flatten().astype(float)
-    if data.size == 0:
+    if bucket_count == 0:
         histogram_buckets = np.array([]).reshape((0, 3))
+    elif data.size == 0:
+        histogram_buckets = np.zeros((bucket_count, 3))
```

Comment:
Optional, but this implementation also handles the 0-bucket-count case, so you could combine these two conditions into just

```py
if bucket_count == 0 or data.size == 0:
    histogram_buckets = np.zeros((bucket_count, 3))
```

Reply:
Done. Thanks!